lib/cocoapods/generator/xcconfig/xcconfig_helper.rb

```diff
@@ -183,17 +183,12 @@ def self.add_language_specific_settings(target, xcconfig)
         #
         # @return [void]
         #
-        def self.add_developers_frameworks_if_needed(xcconfig, platform)
+        def self.add_developers_frameworks_if_needed(xcconfig)
```

Comment:
Doesn't look like it, apparently the second parameter was never documented. Or do you think we should replace "SenTestingKit" with "XCTest" in there?